src/styles/globals.css

```diff
@@ -7,3 +7,7 @@ html,
 body {
 	height: 100%;
 }
+.debug{
```

Comment:
is this class necessary? what's wrong with writing the tailwind in the relevant place?

Reply:
it is just for checkings it will go when we dont need it